packages/st2bundle/install_rhel6.md

```diff
@@ -0,0 +1,48 @@
+## Install IUS repo
+sudo yum install -y https://dl.iuscommunity.org/pub/ius/stable/Redhat/6/x86_64/ius-release-1.0-14.ius.el6.noarch.rpm
+
+# Install mongodb
+sudo yum install -y mongodb mongodb-server
+
+# Install rabbitmq
+sudo rpm --import https://www.rabbitmq.com/rabbitmq-signing-key-public.asc
+curl -sS -k -o /tmp/rabbitmq-server.rpm https://www.rabbitmq.com/releases/rabbitmq-server/v3.3.5/rabbitmq-server-3.3.5-1.noarch.rpm
+sudo yum localinstall -y /tmp/rabbitmq-server.rpm
```

Comment:
Why required specific version of rabbit? It would be cool if in the docs on the step like this the was a remark: *"Grabbing from repo, since ius and os repo provide outdated one (required >= 3.3)"*